packages/connect-explorer/src/actions/trezorConnectActions.ts

```diff
@@ -143,6 +145,7 @@ export const init =
         // Get default coreMode from URL params (?core-mode=auto)
         const urlParams = new URLSearchParams(window.location.search);
         const coreMode = (urlParams.get('core-mode') as ConnectOptions['coreMode']) || 'auto';
+        const deeplink = urlParams.get('deeplink') === 'true'; // TODO
```

Comment:
I imagine, that in the end, deeplink will be only one of the core modes maybe (only available if some conditions are met, such as registering callbacks etc).